examples/poyo/configs/base.yaml

```diff
@@ -0,0 +1,48 @@
+# Base config for training poyo_plus
+
+data_root: ./data/processed/
+log_dir: ./logs
+
+sequence_length: 1.0  # in seconds
+latent_step: 0.125  # in seconds
+
+modality_name: cursor_velocity_2d
+metric:
+  _target_: torchmetrics.R2Score
+  num_outputs: 2
+
+epochs: 1000  # if -1, will use steps
+steps: -1  # if -1, will use epochs. Epochs take precedence.
+eval_epochs: 1
+
+batch_size: 128
+eval_batch_size: null  # if null, will use batch_size
+num_workers: 4
+seed: 42
+
+subtask_weights:
+  - 1.0  # RANDOM
+  - 0.1  # HOLD
+  - 5.0  # REACH
+  - 1.0  # RETURN
+  - 0.1  # INVALID
+  - 0.0  # OUTLIER
```

Comment:
This is hacky. Will clean once the interval-based movement phases are merged into brainsets.